theme/lumo/vaadin-date-time-picker-styles.html

```diff
@@ -16,6 +16,17 @@
         border-top-left-radius: 0;
         border-bottom-left-radius: 0;
       }
+
+      :not(*):placeholder-shown, /* to prevent broken styles on IE */
+      :host([dir="rtl"]) [part="value"]:placeholder-shown,
+      :host([dir="rtl"]) [part="input-field"] ::slotted(input:placeholder-shown) {
+        --_lumo-text-field-overflow-mask-image: none;
+      }
+
+      :host([dir="rtl"]) [part="value"],
+      :host([dir="rtl"]) [part="input-field"] ::slotted(input) {
+        --_lumo-text-field-overflow-mask-image: linear-gradient(to left, transparent, #000 1.25em);
+      }
```

Comment:
I guess these can also be removed from here if this element is made to extend `vaadin-date-picker-text-field`?